react/hooks/useFacetNavigation.js

```diff
@@ -158,7 +159,11 @@ export const buildNewQueryMap = (
   let querySegments = selectedFacets.map(facet => facet.value)
   let mapSegments = selectedFacets.map(facet => facet.map)
 
-  const { ft: fullText, productClusterIds: collection } = fullTextAndCollection
+  const {
+    ft: fullText,
+    productClusterIds: collection,
+    seller,
+  } = fullTextSellerAndCollection
```

Comment:
There is no `fullTextSellerAndCollection ` declared in the code. It is throwing an error

![image](https://user-images.githubusercontent.com/40380674/92517130-90ac2c80-f1ec-11ea-900d-203d2b64a954.png)